src/models/event.ts

```diff
@@ -407,12 +405,6 @@ export class MatrixEvent extends TypedEventEmitter<MatrixEventEmittedEvents, Mat
      */
     public forwardLooking = true;
 
-    /* If the event is a `m.key.verification.request` (or to_device `m.key.verification.start`) event,
-     * `Crypto` will set this the `VerificationRequest` for the event
-     * so it can be easily accessed from the timeline.
-     */
-    public verificationRequest?: VerificationRequest;
```

Comment:
This wasn't deprecated?

Reply:
https://github.com/matrix-org/matrix-js-sdk/blob/7d687533e753f02166a3014dcbddce07a8d1a4eb/src/crypto/verification/request/VerificationRequest.ts#L76-L83

`VerificationRequest` is deprecated and only supported in the legacy crypto. When removing it, these methods are basically dead code.

Reply:
Things should get a deprecation notice before removal